src/components/LatestArticles.vue

```diff
@@ -13,11 +13,16 @@
             <div class="col-md-12 col-lg-6">
                 <router-link :to="articlesData[0].link">
                     <img
+                        v-if="articlesData.imageLink"
```

Comment:
I am sorry @fluffynuts  - copy-paste error :/ there should be the iterator to point to the first article
```suggestion
                        v-if="articlesData[0].imageLink"
```

Reply:
I have created a pull request on that #24.